bin/include/build-classpath.sh

```diff
@@ -42,6 +42,9 @@ case "`uname`" in
 esac
 
 includeToClassPath() {
+    SAVEIFS=$IFS
```

Comment:
Can it be replaced with following construction?
```
while IFS=$(echo -en "\n\b") read file
do
    ...
done < $1/*
```

Reply:
Maybe, but why? It adds one more level of nesting. This is not good.

Reply:
There will be no additional nesting, as I suggest to replace `for` loop with `while` loop, because `while` loop can use temporary IFS override, i.e.:
```
while IFS=$(echo -en "\n\b") read file
do
    if [ -d ${file} ] && [ -d "${file}/target" ]; then
        ...
    fi
done < $1/*
```

Reply:
Ok, it can be done, but your example doesn't work. Maybe we just leave my option?